docs/Update-SSHKeys.md

```diff
@@ -22,6 +22,37 @@ You will need the following information for the MachineConfig that will be used
 
 - `sshAuthorizedKeys:` you will need one or more public keys to be assigned to user `core`.  Multiple SSH Keys should begin on different lines and each be preceded by `-`.
 
+## Creating SSH for workers
+
+In case your cluster has been installed without providing the ssh key for the `core` user at installation time, you can easily create a MachineConfig that will ship the key to the nodes in a pool:
```

Comment:
i'd add:
"installation time (verify that `99-worker-ssh`/`99-master-ssh` are missing, by checking `oc get machineconfigs`),